pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/EventService.java

```diff
@@ -96,8 +98,11 @@ protected static boolean triggerDssByKitEvent(KitDDPNotification kitDDPNotificat
     @VisibleForTesting
     protected static boolean triggerDssWithEvent(@NonNull String eventType, DDPInstance ddpInstance, long eventDate,
                                         @NotNull String ddpParticipantId, @NotNull String eventInfo, KitReasonType reason) {
-        final long initialInterval = 500; // base delay in milliseconds
-        final double multiplier = 2.0; // exponential backoff multiplier
+
+        final long initialInterval = DSMConfig.getIntFromConfig(ApplicationConfigConstants.EVENT_RETRY_INTERVAL_MS); // base delay in
```

Comment:
looks like you might get null if not in conf file. 
May be default to some valid int value if not in path? 
same for multiplier

Reply:
oh great catch! will fix, Thx

Reply:
I think it's safe to fail if it's not in the config.  With this change, we expect the value to be in the config, so it's okay to leave as is IMO.